db/seeds.rb

```diff
@@ -6,7 +6,7 @@
 FactoryGirl.create(:team, :last_season, kind: "sponsored")
 FactoryGirl.create(:team, :last_season, kind: "voluntary")
 
-# Users with different roles
+# # Users with different roles
```

Comment:
It's a feature ;-) Like an H2, under H1 Users. But maybe MaudFlavoredMarkdown doesn't work that good if I have to explain it. Will delete it.